docs/DeveloperGuide.md

```diff
@@ -292,6 +292,51 @@ Aspect: What fields should the `find` command search through?
   - Pros: Easy to implement, with minimal modification to existing tests.
   - Cons: Limited breadth of search, does not align with definition of unique application. 
 
+### Sort Feature
+
+#### Implementation
+
+The sort feature allows the user to sort the application list using company names, positions, application dates or interview dates. Each of these orders can also be reversed.
+
+The class diagram below shows the classes in the Logic component relevant for sorting:
+
+![Sort Class Diagram](images/SortClassDiagram.png)
+
+There is an abstract `SortCommand` class that inherits from the abstract `Command` class. Then, there is a concrete `SortCommand` subclass for each possible order of sort. Meanwhile, there is a single `SortCommandParser` class. When it parses the arguments supplied to a `sort` command, it decides which of the `SortCommand` subclasses to instantiate.
+
+The following sequence diagram shows the operation of the command `sort o/date r/`:
```

Comment:
Not sure if this is repetitive? Since above the developer guide, the sequence diagram of delete diagram has been drawn and both are quite similar 🤔

Reply:
Oh yeah... good point. But that does give me an idea on something more meaningful to show. I'll go add in now